### PR TITLE
feat(statsd) update StatsD document

### PR DESCRIPTION
**Pull request by vm-001**
### Summary

1. Update StatsD document based on the latest code
2. Add a message about removing StatsD Advanced plugin

### CODE REVIEW

File: app/_hub/kong-inc/statsd/_index.md [hunk 1]
```diff
@@ -119,6 +110,24 @@ params:
...
+{% if_plugin_version gte:3.0.x %}
+Metric                     | Description | Namespace
+---                        | ---         | ---
+`request_count`            | The number of requests. | `kong.service.<service_identifier>.request.count`
+`request_size`             | The request's body size in bytes. | `kong.service.<service_identifier>.request.size`
+`response_size`            | The response's body size in bytes. | `kong.service.<service_identifier>.response.size`
+`latency`                  | The time interval in milliseconds between the request and response. | `kong.service.<service_identifier>.latency`
+`status_count`             | Tracks each status code returned in a response. | `kong.service.<service_identifier>.status.<status>`
+`unique_users`             | Tracks unique users who made requests to the underlying Service or Route. | `kong.service.<service_identifier>.user.uniques`
+`request_per_user`         | Tracks the request count per Consumer. | `kong.service.<service_identifier>.user.<consumer_identifier>.request.count`
```
Comment (Guaris) [inline app/_hub/kong-inc/statsd/_index.md:122]:
```suggestion
`request_per_user`         | Tracks the request count per consumer. | `kong.service.<service_identifier>.user.<consumer_identifier>.request.count`
```

File: app/_hub/kong-inc/statsd/_index.md [hunk 2]
```diff
@@ -119,6 +110,24 @@ params:
...
 
+{% if_plugin_version gte:3.0.x %}
+Metric                     | Description | Namespace
+---                        | ---         | ---
+`request_count`            | The number of requests. | `kong.service.<service_identifier>.request.count`
+`request_size`             | The request's body size in bytes. | `kong.service.<service_identifier>.request.size`
+`response_size`            | The response's body size in bytes. | `kong.service.<service_identifier>.response.size`
+`latency`                  | The time interval in milliseconds between the request and response. | `kong.service.<service_identifier>.latency`
+`status_count`             | Tracks each status code returned in a response. | `kong.service.<service_identifier>.status.<status>`
+`unique_users`             | Tracks unique users who made requests to the underlying Service or Route. | `kong.service.<service_identifier>.user.uniques`
```
Comment (Guaris) [inline app/_hub/kong-inc/statsd/_index.md:121]:
```suggestion
`unique_users`             | Tracks unique users who made requests to the underlying Service or route. | `kong.service.<service_identifier>.user.uniques`
```

File: app/_hub/kong-inc/statsd/_index.md [hunk 3]
```diff
@@ -119,6 +110,24 @@ params:
...
+`request_per_user`         | Tracks the request count per Consumer. | `kong.service.<service_identifier>.user.<consumer_identifier>.request.count`
+`upstream_latency`         | Tracks the time in milliseconds it took for the final Service to process the request. | `kong.service.<service_identifier>.upstream_latency`
+`kong_latency`             | Tracks the internal Kong latency in milliseconds that it took to run all the plugins. | `kong.service.<service_identifier>.kong_latency`
+`status_count_per_user`    | Tracks the status code for per Consumer per Service. | `kong.service.<service_identifier>.user.<consumer_identifier>.status.<status>`
```
Comment (Guaris) [inline app/_hub/kong-inc/statsd/_index.md:125]:
```suggestion
`status_count_per_user`    | Tracks the status code per consumer per service. | `kong.service.<service_identifier>.user.<consumer_identifier>.status.<status>`
```

File: app/_hub/kong-inc/statsd/_index.md [hunk 4]
```diff
@@ -119,6 +110,24 @@ params:
...
+`status_count_per_workspace`         | The status code per workspace. | `kong.service.<service_identifier>.workspace.<workspace_identifier>.status.<status>`
+`status_count_per_user_per_route`    | The status code per Consumer per Route. | `kong.route.<route_id>.user.<consumer_identifier>.status.<status>`
```
Comment (Guaris) [inline app/_hub/kong-inc/statsd/_index.md:127]:
```suggestion
`status_count_per_user_per_route`    | The status code per consumer per route. | `kong.route.<route_id>.user.<consumer_identifier>.status.<status>`
```

File: app/_hub/kong-inc/statsd/_index.md [hunk 5]
```diff
@@ -192,7 +197,13 @@ Field         | Description                                             | Dataty
 ### {{site.base_gateway}} 3.0.x
 
 * Merged features of the StatsD Advanced plugin into the StatsD plugin. The StatsD plugin now includes the following:
-  * New parameters for StatsD: `hostname_in_prefix`, `udp_packet_size`, `ues_tcp`, `allow_status_codes`, `consumer_identifier_default`, `service_identifier_default`, `workspace_identifier_default`, `service_prefix_in_metric`.
+  * New parameters for StatsD: `hostname_in_prefix`, `udp_packet_size`, `ues_tcp`, `allow_status_codes`, `consumer_identifier_default`, `service_identifier_default`, `workspace_identifier_default`.
   * New metrics: `status_count_per_workspace`, `status_count_per_user_per_route`, `shdict_usage`
   * New metric fields: `service_identifier`, `workspace_identifier`
+
+* Breaking Changes
```
Comment (Guaris) [inline app/_hub/kong-inc/statsd/_index.md:204]:
```suggestion
* Breaking changes
```

File: app/_hub/kong-inc/statsd/_index.md [hunk 6]
```diff
@@ -192,7 +197,13 @@ Field         | Description                                             | Dataty
...
+  * The metric name that related with the Service has been renamed by adding a `service.` prefix. e.g. `kong.service.<service_identifier>.request.count`
```
Comment (Guaris) [inline app/_hub/kong-inc/statsd/_index.md:205]:
```suggestion
  * The metric name that is related to the service has been renamed by adding a `service.` prefix. e.g. `kong.service.<service_identifier>.request.count`
```

File: app/_hub/kong-inc/statsd/_index.md [hunk 7]
```diff
@@ -192,7 +197,13 @@ Field         | Description                                             | Dataty
...
+  * The metric `kong.<service_identifier>.request.status.<status>.count` from Metrics `status_count` and `status_count_per_user` has been renamed to `kong.service.<service_identifier>.status.<status>.count`
+  * The metric `*.status.<status>.total` from Metrics `status_count` and `status_count_per_user` has been removed.
```
Comment (Guaris) [inline app/_hub/kong-inc/statsd/_index.md:207]:
```suggestion
  * The metric `*.status.<status>.total` from metrics `status_count` and `status_count_per_user` has been removed.
```

**Review by Guaris — APPROVED** [review]

**Comment by vm-001** [comment]
https://github.com/Kong/docs.konghq.com/pull/4265/files#diff-584bc962f3a8d17866d3bd6cd3c982b0aaf600a1735800832f2e1d8f190aa79bR13-R15

I'm not sure if this message type is appropriate. Need to be checked @Guaris

**Comment by Guaris** [comment]
I rewrote some of the copy on that banner but it looks good to me.

**Pull request merged**